android/src/main/kotlin/com/abilia/system_settings_editor/SystemSettingsEditorPlugin.kt

```diff
@@ -163,4 +87,19 @@ class SystemSettingsEditorPlugin : FlutterPlugin, MethodCallHandler, ActivityAwa
   override fun onDetachedFromActivity() {
     this.activity = null
   }
+
+  companion object{
+
+    const val METHOD_GET_BRIGHTNESS = "getBrightness"
```

Comment:
I don't really see the use of these constants. They will only be used in onMethodCall so not really any extra "spelling safety" and a bit harder to map the functionName to the handler when trying to follow the code.

Reply:
No, you're right. I created them with the intention of using them across the plugin, but it's of course not possible to use them in the dart section. I did some googling to see if there was any way to achieve it, but didn't find any that were not cumbersome.